packaging/package/package.py

```diff
@@ -20,7 +21,10 @@ def main():
         ConnectorFile("connectionResolver.tdr", "connection-resolver")]
 
     validate_xsd(files_to_package, path_from_args)
```

Comment:
I think we should make the packaging only happen if validate_xsd returns true. Otherwise, we may try to package a plugin we know is broken.

Reply:
ok sounds good.